lib/src/main/java/com/futurewei/alcor/common/utils/SpringContextUtil.java

```diff
@@ -13,7 +13,7 @@
         See the License for the specific language governing permissions and
         limitations under the License.
 */
-package com.futurewei.alcor.portmanager.util;
+package com.futurewei.alcor.common.utils;
```

Comment:
Move util class from port manager to common is great. I remember previous attempt of moving this caused some UT failure or so.